packages/@angular/cli/commands/help.ts

```diff
@@ -56,7 +56,7 @@ const HelpCommand = Command.extend({
 
       if (rawArgs.length > 0) {
         if (cmd === rawArgs[0]) {
-          this.ui.writeLine(command.printDetailedHelp(commandOptions));
+          this.ui.writeLine(command.printBasicHelp(commandOptions));
```

Comment:
Yes, the bug we were solving was regarding blueprints.

Here's the PR that changed it: https://github.com/angular/angular-cli/pull/4267

So basically if there's detailed help, show that. If not, show basic.